packages/core/src/lib/GooglePlay.ts

```diff
@@ -48,6 +53,20 @@ export class GooglePlay {
     }
   }
 
+  /**
+   * Generates an editId that should be used in all play operations.
+   * @param packageName - The package that will be worked upon.
+   * @param operation - The Play Operation which requires an editId injected
+   * @returns - A PlayOperationResult which contains the output of the operation in a field that was worked upon.
+   */
+  async performPlayOperation(packageName: string,
```

Comment:
I think I got this worked out the way we want it.

One note, rather than write the generic function invocation like this:
```
await this.googlePlay.performPlayOperation<number>(...
```

I wrote it like this
```
await this.googlePlay.performPlayOperation(...
```

This okay?

Reply:
Yeah, if the compiler can figure it out and it's obvious to anyone reading the code, it's fine to leave this out.